controllers/authControllerTestController.test.js

```diff
@@ -0,0 +1,56 @@
+import { testController } from './authController.js';
+
+
+// Test suite for testController
+describe('testController', () => {
+    let req;
+    let consoleLogSpy;
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+        req = {};
+
+        //Reinitialise console log spy
+        consoleLogSpy = jest.spyOn(console, 'log').mockImplementation(() => {});
+    });
+
+    afterEach(() => {
+        //Restore original functionality of console.log
+        consoleLogSpy.mockRestore();
+    });
+
+
+    it('should return protected routes if there is no error', async () => {
```

Comment:
Test does not need to be async as the function is not async, same for the test below

Reply:
Okay thank you!